README.md

```diff
@@ -85,6 +85,11 @@ $ sudo rpm -Uvh gdrcopy-<version>.<platform>.rpm
 $ sudo rpm -Uvh gdrcopy-devel-<version>.<platform>.rpm
 ```
 
+Non-DKMS kmod package can be generated by adding `-m` option to `build-rpm-packages.sh`.
```

Comment:
Can you rephrase here? Let's use the convention in https://forums.fedoraforum.org/showthread.php?249991-kmod-akmod-dkms, i.e. kmod and dkms are distinct ways to deliver kernel modules.

Can you also clarify that build-rpm-packages.sh builds dkms by default?

Reply:
also, for line 77, can you specify that it is required on RPM based distro when building dkms ?
and consider moving that EPEL extra installation step to the requirements section.